aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1141,16 +1141,6 @@ module aptos_framework::stake {
         staking_config: &StakingConfig,
     ) acquires StakePool, AptosCoinCapabilities, ValidatorConfig, ValidatorFees {
         let stake_pool = borrow_global_mut<StakePool>(pool_address);
-
-        // First, distribute transaction fees.
```

Comment:
PR desc doesn't explain why this is necessary. Was this causing issues?

Reply:
Yes, it was - updated the PR description!